### ISSUE TITLE
bug in danger running imported Dangerfile

### ISSUE THREAD

**Issue by samdmarshall** [danger/danger#487]
I came across this issue today while trying to run Danger on a new PR for one of my projects, while the Dangerfile in the project repo has been updated for the 3.0 syntax, I don't know if the org-level Dangerfile has. 

```
Running danger (PR)... 
/tmp/d20160829-19865-1c6uqzd/temporary_danger.rb:2:in `<top (required)>':  (Danger::DSLError)
[!] Invalid `Dangerfile` file: undefined local variable or method `git' for main:Object. Updating the Danger gem might fix the issue.

 #  from Dangerfile:30
 #  -------------------------------------------
 #  # Run the shared Dangerfile with these settings
 >  danger.import_dangerfile "samdmarshall/danger"
 #  -------------------------------------------
	from /opt/circleci/ruby/ruby-2.2.4/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /opt/circleci/ruby/ruby-2.2.4/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/plugins/dangerfile_danger_plugin.rb:116:in `block in import_local'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/plugins/dangerfile_danger_plugin.rb:114:in `each'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/plugins/dangerfile_danger_plugin.rb:114:in `import_local'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/plugins/dangerfile_danger_plugin.rb:103:in `import_url'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/plugins/dangerfile_danger_plugin.rb:63:in `import_dangerfile'
	from Dangerfile:30:in `block in parse'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/dangerfile.rb:194:in `eval'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/dangerfile.rb:194:in `block in parse'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/dangerfile.rb:190:in `instance_eval'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/dangerfile.rb:190:in `parse'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/danger_core/executor.rb:42:in `run'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/lib/danger/commands/runner.rb:56:in `run'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/gems/danger-3.0.3/bin/danger:5:in `<top (required)>'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/bin/danger:23:in `load'
	from /opt/circleci/.rvm/gems/ruby-2.2.4/bin/danger:23:in `<main>'
```

For details see: https://circleci.com/gh/samdmarshall/pyconfig/291

Org-level Dangerfile: https://github.com/samdmarshall/danger/blob/master/Dangerfile

**Comment by jk** [comment]
@orta So this issue can be closed?

**Comment by orta** [comment]
Yeah 👯 - fixed own 3.1.1
